04.ls/ls.rb

```diff
@@ -1,6 +1,20 @@
 #!/usr/bin/env ruby
 
-dir = Dir.glob '*'
+require 'optparse'
+
+option = nil
+ARGV.options do |opt|
+  opt.on('-a') { option = '-a' }
+  opt.parse!
+end
+
+case option
+when '-a'
+  dir = Dir.glob('*', File::FNM_DOTMATCH)
```

Comment:
`dir = Dir.glob('*'` のコードが重複していますね。
オプションの有無で第2引数のみを切り替えるようにすると、重複をなくせるはずなので、チャレンジしてみていただけますでしょうか？ 🙏

Reply:
```suggestion
second_argument = option == '-a' ? File::FNM_DOTMATCH : nil.to_i
dir = Dir.glob('*', second_argument)
```
上記のように修正してみたので、確認お願いいたします。